app/src/main/res/layout/list_steps.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
         xmlns:android="http://schemas.android.com/apk/res/android"
-        android:layout_width="match_parent"
+        xmlns:app="http://schemas.android.com/apk/res-auto" android:layout_width="match_parent"
         android:orientation="horizontal"
+        android:padding="@dimen/standard_margin"
         android:layout_height="wrap_content">
 
     <TextView
             android:id="@+id/txt_short_description"
             android:padding="@dimen/standard_padding"
             android:layout_width="0dp"
+            android:textSize="16sp"
```

Comment:
@KhinThiriSoe textSize

Reply:
<img width="636" alt="Screen Shot 2019-03-12 at 1 04 52 PM" src="https://user-images.githubusercontent.com/10254643/54178186-7c885180-44c7-11e9-9873-21324d753038.png">

Reply:
@KhinThiriSoe that's warning is for layout_width. I'm talking about textSize

Reply:
app:tabIndicatorHeight = 0dp I was wrong writing comment

Reply:
@KhinThiriSoe no, you're still wrong. the photo you added is for this `txt_short_description` and it's for `layout_width`

Let's talk about `app:tabIndicatorHeight` in a different thread